include/etiss/SimpleMemSystem.h

```diff
@@ -100,34 +107,32 @@ class MemSegment
         else
         {
             mem_ = new etiss::uint8[size];
-            memInit(initVal);
+            memInit(initString);
             self_allocated_ = true;
         }
     }
 
     // Can be overwritten afterwards with load_elf
-    void memInit(uint8_t initVal = 0)
+    void memInit(std::string initString = 0)
```

Comment:
std::string default argument should be `= ""` or removed since it now will be managed by etiss::initialize